lib/Service/PageService.php

```diff
@@ -249,10 +249,12 @@ private function notifyPush(int $collectiveId, string $userId): void {
 		]);
 	}
 
-	private function updatePage(int $collectiveId, int $fileId, string $userId, ?string $emoji = null, ?string $subpageOrder = null): void {
+	private function updatePage(int $collectiveId, int $fileId, string $userId, ?string $emoji = null, ?string $subpageOrder = null, bool $updateEditUser = true): void {
```

Comment:
That's a long list of arguments. I'd define a separate function for updating the subpage order instead. That way you'd have two functions with 4 arguments rather than one with 6.
As far as i can tell we never update the subpage order and the emoji at the same time. We also never update the last user when updating the subpage order etc. - It's nice to have a flexible and powerful function when you need it. Here it feels like you never need the added complexity it brings.

Reply:
Good point, thanks. Refactored now.